controller/ddstypes/ControllerCommand/ControllerCommand.idl

```diff
@@ -0,0 +1,5 @@
+struct ControllerCommand
+{
+    string command;
```

Comment:
Maybe we can add comments about the possible values here?
If not here, somewhere else in Controller.py